README.md

```diff
@@ -235,15 +235,10 @@ publish.standard();
                 <code>cleanCmd</code>
             </td>
             <td>
-                The CLI to execute which cleans up any temporary changes to the package.json file.
-                <ul>
-                    <li>
-                        <code>${package}</code> will be substituted with the path to the executing modules package.json file.
-                    </li>
-                </ul>
+                The CLI to execute which cleans up any temporary changes to the package.json and package-lock.json files.
             </td>
             <td>
-                "git checkout -- ${package}"
+                "git checkout -- package.json package-lock.json"
```

Comment:
Needs to be updated to the new value, and need documentation for other new options.